src/app/features/ahbs/components/export-button/export-button.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input } from '@angular/core';
+import { AhbService } from '../../../../core/api';
+
+@Component({
+  selector: 'app-export-button',
+  standalone: true,
+  templateUrl: './export-button.component.html',
+})
+export class ExportButtonComponent {
+  @Input() formatVersion!: string;
+  @Input() pruefi!: string;
+
+  constructor(private ahbService: AhbService) {}
+
+  onClickExport(): void {
+    this.ahbService
+      .getAhb$VndOpenxmlformatsOfficedocumentSpreadsheetmlSheet({
+        'format-version': this.formatVersion,
+        pruefi: this.pruefi,
+        format: 'xlsx',
+      })
+      .subscribe({
+        next: (blob: Blob) => {
+          const url = window.URL.createObjectURL(blob);
+          const link = document.createElement('a');
+          link.href = url;
+          link.download = `AHB_${this.formatVersion}_${this.pruefi}.xlsx`;
+          link.click();
+          window.URL.revokeObjectURL(url);
+        },
+        error: (error) => {
+          console.error('error downloading XLSX:', error);
+        },
+      });
+  }
```

Comment:
```suggestion
  async onClickExport(): Promise<void> {
    const blob = firstValueFrom(this.ahbService
      .getAhb$VndOpenxmlformatsOfficedocumentSpreadsheetmlSheet({
        'format-version': this.formatVersion,
        pruefi: this.pruefi,
        format: 'xlsx',
      }));
      const url = URL.createObjectURL(blob);
      const link = document.createElement('a');
      link.href = url;
      link.download = `AHB_${this.formatVersion}_${this.pruefi}.xlsx`;
        link.click();
        URL.revokeObjectURL(url);
  }
```
Suggestion: use Promises instead of Observables for more readable and consistent code